src/mlpack/core/data/imputation_methods/custom_imputation.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file custom_imputation.hpp
+ * @author Keon Kim
+ *
+ * Definition and Implementation of the empty CustomImputation class.
+ */
+#ifndef MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+#define MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+
+#include <mlpack/core.hpp>
+#include <cmath>
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+template <typename T>
+class CustomImputation
```

Comment:
If you can provide some comments on what this class is and what it does (also for the `Impute()` function), I think it would be really helpful for users.